src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
enums should not be named in plural form 
```suggestion
public enum Color {
```